src/phpdoc/type_parser.go

```diff
@@ -49,6 +49,14 @@ const (
 	// Examples: `int` `\Foo\Bar` `$this`
 	ExprName
 
+	// ExprClass is a type of class that is identified by its name.
+	// Examples: `\Foo` `\Bar`
+	ExprClass
+
+	// ExprMember is a type of class member that is identified by its name.
+	// Examples: `\Foo` `\Bar`
```

Comment:
Examples look like a copy/paste from the ExprClass. I believe they should be at least a little different, as it's unusual to have a class as a member of another class (not sure it's possible, but I might be wrong).

Reply:
This is a copy-paste error